client/fresheyes-web/src/app/components/form/FormSection.tsx

```diff
@@ -28,15 +28,20 @@ const FormSection = ({ username }: { username: string | null | undefined }) => {
 
   const { owner, repo, pull_number } = formValues;
 
-  const processPullRequest = async (clickable: boolean, clickableArgs: PullRequest) => {
+  const processPullRequest = async (e: React.FormEvent) => {
+    e.preventDefault()
     setLink("");
     setError("");
     setLoading({ loader: false, modal: false, isInstalledModal: false });
 
-    if (pull_number === 0 && clickable === false) {
+    if (pull_number === 0 || !pull_number) {
       alert("You must pass a number that is not zero");
       return;
     }
+    if (!owner  || !repo) {
```

Comment:
Why not just make the html input fields required instead of this extra logic? We still need the pull_number check though since we want to disallow zero values

Reply:
Yes that can be done